examples/migration/campaign_management/create_complete_campaign_adwords_api_only.rb

```diff
@@ -23,9 +23,9 @@
 # campaign budget, a Search campaign, ad groups, keywords and expanded text ads. None of the
 # functionality has yet been migrated to the Google Ads API.
 
-
-
+require 'google/ads/google_ads'
 require 'adwords_api'
+require 'optparse'
```

Comment:
I don't think you need google/ads/google_ads or optparse for the AdWords API only version. Are you including them just for consistency with the other ones?